cmd/courier/main.go

```diff
@@ -109,6 +138,31 @@ func status(c *cli.Context) (err error) {
 	return printJSON(rep)
 }
 
+// Get a secret from the secret manager.
+func getSecret(c *cli.Context) (err error) {
+	conf := config.SecretsConfig{
+		Enabled:     true,
+		Project:     c.String("project"),
+		Credentials: c.String("credentials"),
+	}
+
+	secrets, err := secrets.NewClient(conf)
+	if err != nil {
+		return cli.Exit(err, 1)
+	}
+
+	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
+	defer cancel()
+
+	var secret []byte
+	if secret, err = secrets.GetLatestVersion(ctx, c.String("name")); err != nil {
+		return cli.Exit(err, 1)
+	}
+
+	fmt.Println(string(secret))
```

Comment:
Printing the secret was kind of the point since this is in the CLI, this code won't be called from the service and it's more of a debug thing that I wrote to quickly test if the secret manager client was working.

Reply:
got it - I had figured you added the print statement to test the functionality but may have forgotten to remove it - thanks for confirming!